### DISCUSSION TITLE
useState and input

### DISCUSSION THREAD

**Discussion by FreytagDom** [neuefische/questions#155] [Web] [answered]
Hallo ihr lieben, ich komme einfach nicht weiter.
Zum Problem:
Ich habe ein Input field und möchte dies nach der Eingabe von Daten in einer Karte angezeigt und gespeichert werden.
Jetzt habe ich es soweit das ich keine Fehler mehr bekomme aber mir wird nichts angezeigt oder übergeben, und ich weiß einfach nicht was ich grade falsch mache oder wo der Fehler im Code liegt. 
Hoffe einer kann mir weiter helfen.

Danke schon einmal :)
![Bildschirmfoto 2022-09-21 um 13 52 21](https://user-images.githubusercontent.com/109357762/191497138-69444109-9b0a-4a44-a9e4-5844e01237f6.png)
![Bildschirmfoto 2022-09-21 um 13 52 33](https://user-images.githubusercontent.com/109357762/191497176-b20ecc66-57fc-4ace-8033-8fbb972bb25d.png)
![Bildschirmfoto 2022-09-21 um 13 52 48](https://user-images.githubusercontent.com/109357762/191497201-000e0605-3f5b-4895-adf4-08de69d6ab15.png)

**Comment by NikRoe** [comment]
Hey Dominic, 
danke für die Screenshots! Die helfen bereits weiter.

Kannst du uns noch einen Link zum Pull Request schicken?

Danke dir!

**Comment by FreytagDom** [comment]
https://github.com/FreytagDom/capstone-project/pull/6

**Comment by NikRoe** [comment, marked as answer]
Danke dir!

Wir haben ein paar Änderungen vorgenommen und einen Pull Request gegen deinen Pull Request gestellt, wo du die Änderungen detailliert nachverfolgen kannst, die wir vorgenommen haben [hier ist der PR](https://github.com/FreytagDom/capstone-project/pull/7/files).

Das Grundproblem war, dass du der Componente deines Formulars nicht die Funktion weitergegeben hast, die den State in der `index.js` updaten soll. Zudem hast du diese in dem Formular nicht entgegengenommen. Die wichtigsten Änderungen haben wir dir hier hervorgehoben:

![Screenshot 2022-09-21 at 15 08 37](https://user-images.githubusercontent.com/98757391/191512571-12e9a046-c220-49df-a8b5-454cf98cda6e.png)

Wir haben zudem ein paar Kleinigkeiten ausgebessert und Dinge mit Kommentaren versehen bzw. gänzlich auskommentiert, wenn die Inhalte keine Relevanz oder Funktion hatten.

**Comment by FreytagDom** [comment]
Vielen lieben Danke.
Da war ich am Anfang ja schon recht dicht dran
